.github/workflows/test-upstream.yml

```diff
@@ -173,59 +193,32 @@ jobs:
 
   report-failures:
     name: Open issue for upstream dev failures
-    needs: [test-dev, cluster-dev]
+    needs: [test-dev, cluster-dev, import-dev]
     if: |
       always()
       && (
-        needs.test-dev.result == 'failure' || needs.cluster-dev.result == 'failure'
+        needs.test-dev.result == 'failure'
+        || needs.cluster-dev.result == 'failure'
+        || needs.import-dev.result == 'failure'
       )
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Report failures
-        uses: actions/github-script@v3
+      - uses: actions/download-artifact@v3
+      - name: Prepare logs & issue label
+        run: |
+          # import testing doesn't generate a pytest log but one is needed for xarray-contrib/issue-from-pytest-log
+          if ls test*.jsonl 1> /dev/null 2>&1; then
+            paste -d"\n" test*.jsonl > results.jsonl
+          else
+            touch results.jsonl
+          fi
```

Comment:
It's not immediately obvious to me how `xarray-contrib/issue-from-pytest-log` will handle an empty JSONL file, I suppose we could fill it with some arbitrary row if needed but ideally we could explore extending the action such that the log path is an option argument.